public/assets/js/monster/fields.js

```diff
@@ -45,7 +45,7 @@ var monsterFields = [
     'base64_image', // base64_image PRO
     'wysiwyg', // ckeditor PRO
     'color_picker', // color_picker PRO
-    'start_date', 'end_date', // date_range PRO
+    'start_date', //'end_date', // date_range PRO
```

Comment:
The field stands by 'start_date' only (either in the current vanilla JS solution, or the old one with jQuery).
We either change the query selector way to work, or we change the field.